train.py

```diff
@@ -2,7 +2,10 @@
 import difflib
 import importlib
 import os
+import time
 import uuid
+from ast import Raise
```

Comment:
why is that needed? ast is for analysing code, no?

Reply:
So weird. I think VScode auto imported it...